frida-gum/src/interceptor/invocation_listener.rs

```diff
@@ -113,6 +115,11 @@ impl<'a> InvocationContext<'a> {
         unsafe { gum_sys::gum_invocation_context_get_return_address(self.context) as usize }
     }
 
+    /// Get the 'replacement_data' passed at replace time.
+    pub fn replacement_data(&mut self) -> *mut c_void {
```

Comment:
This should return a `NativePointer` not a raw pointer to stay consistent with the rest of the API.

Reply:
I've made it return an `Option<NativePointer>` to be more consistent.